src/app/Dashboard/AddTopic.js

```diff
@@ -26,12 +28,16 @@ class AddTopic extends React.Component {
         {!this.state.isHidden && (
           <form className="add-topic-form" onSubmit={e => this.onSubmit(e)}>
             <label>Topic Name</label>
-            <input type="text" name="folderName" />
+            <input type="text" name="folderTitle" />
```

Comment:
I think the more idiomatic way to handle this would be to create a ref instead of relying on the name attribute. This is because any changes to the name will have the unintentional side-effect of breaking your submit handler, which we don't want! Side effects are bad, mmk?

```js
// For the input: notice the ref
<input ref={input => this.titleInput = input} type="text" name="folderTitle" />
// titleInput can be named anything you want, so feel free to change it.
```

This ref gives us access to the actual element, and we assign it the arbitrary value `this.titleInput`. In other words, we are adding a property to our component instance called `titleInput`, since `this` refers to the component instance.

Now let's access our ref in your event handler.

```js
// in the event handler
let title = this.titleInput.value;
```

Voila! Now we're using React refs instead of the name attribute, so changing the name won't break the event handler.